### ISSUE TITLE
Any plan to add more preprecessors introduced in Controlnet 1.1?

### ISSUE THREAD

**Issue by JoeShu0** [Fannovel16/comfy_controlnet_preprocessors#31]
The controlnet 1.1 has been released, introducing some new models and preprocessors.  Is there any plan to adding these new preprocessor?
Any insight would be greatly appreciated.

**Comment by JoeShu0** [comment]
> I’m adapting them to this repo but due to some works irl it is a bit slow.

Glad hearing this from you, looking forward to it~
Yeah things irl sucks, by all means don't rush. 

Thanks for bringing complete controlnet support to comfyui,.
Wish you a pleasant day~

**Comment by eldragon0** [comment]
You're my hero!

**Comment by Obnyamshi** [comment]
looking forward to lineart_preprecessor, thanks!

**Comment by amir84ferdos** [comment]
> looking forward to lineart_preprecessor, thanks!

yes, it is very useful

**Comment by Fannovel16** [comment]
I added 
* Zoe-DepthMapPreprocessor
* BAE-NormalMapPreprocessor
* New HED (enabled by default as `v1.1` in version option), added safe option to HED and PiDiNet
* Added Openpose preprocessor v1.1, TilePreprocessor
* UniFormer-SemSegPreprocessor (alias of SemSegPreprocessor)
* OneFormer-COCO-SemSegPreprocessor
* OneFormer-ADE20K-SemSegPreprocessor
* LineArtPreprocessor
* AnimeLineArtPreprocessor

**Comment by Fannovel16** [comment]
Not really related but should I add MangaLineExtraction to this repo?
https://github.com/Mikubill/sd-webui-controlnet/issues/1046

**Comment by bropines** [comment]
> It's not really related, but should I add MangaLineExtraction to this repository? [Mikubill/sd-webui-controlnet#1046](https://github.com/Mikubill/sd-webui-controlnet/issues/1046)

Yes. Because waiting for lvmin or mikubill to do this is extremely long. And they ignore me everywhere

**Comment by Fannovel16** [comment]
I just added it as Manga2Anime-LineArtPreprocessor. Note that the output from it will be white lines on black canvas

**Comment by za-wa-n-go** [comment]
Are there any plans to port the shuffle pre-processor?

**Comment by icelandno1** [comment]
@Fannovel16 Please, is it possible to add ***lineart_realistic*** preprocessor, and ***control mode: Balanced\ControlNet is more important*** functions?
These are very versatile, and useful, and very much anticipated, please.

**Comment by Fannovel16** [comment]
@icelandno1 I will add lineart_realistic when I have time. But Control mode can't be added since this repo is only for preprocessors making hint images for CN (canny edge images or stickman for example).
